src/components/BookmarkCard/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Card, Link as UswdsLink } from '@trussworks/react-uswds';
+import classnames from 'classnames';
+
+import Divider from 'components/shared/Divider';
+import SystemHealthIcon from 'components/SystemHealthIcon';
+import { IconStatus } from 'types/iconStatus';
+
+import BookmarkCardIcon from './BookmarkCardIcon';
+
+import './index.scss';
+
+type BookmarkCardProps = {
+  className?: string;
+  type: string;
```

Comment:
What are the different "types" that might be used, besides `"systemList"`? It's not clear from the name what this prop is for (it looks like it's used to identify where to pull i18n'd text from?), and it might be possible to constrain the type more.

Reply:
I added this for future scalability if we want to bookmark other table that have different headers using translation